web/wwwroot/js/site.js

```diff
@@ -17,3 +17,12 @@ function checkIfSelectedValueIsTarget(target){
     }
 }
 
+function copyLinkToClipboard(linkToCopy){
+    // Copy the text passed into the function plus the rest of the url
+    let fullLink = window.location.origin+"/invite/"+linkToCopy
+    navigator.clipboard.writeText(fullLink);
+    
+    // Change the button styling to show the link has been copied
+    document.getElementById("copyBtn").innerHTML = "<i class='bi bi-check'></i> invite link copied"
+    document.getElementById("copyBtn").className = "btn btn-success my-3"

```

Comment:
We could extract the copy button into its own variable here then we wouldn't have to get it twice :)